core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -93,13 +107,80 @@
       "<code>MetricReporter</code> interface allows plugging in classes that will be notified " +
       "of new metric creation. The JmxReporter is always included to register JMX statistics.";
   protected static final String METRICS_REPORTER_CLASSES_DEFAULT = "";
+  public static final String SSL_KEYSTORE_LOCATION_CONFIG = "ssl.keystore.location";
```

Comment:
I expect the number of configs to continue to grow as we flesh out support for things like certificate revocation. The number shouldn't be large, but having to update multiple different places may be a bit error-prone.